### PR TITLE
base: new blocks for the header template

### PR DESCRIPTION
**Pull request by greut**
Adds more blocks to the header template to enable more customizations in the overlays.

Tind is using those.

### CODE REVIEW

File: invenio/base/templates/header_base.html [hunk 1]
```diff
@@ -33,12 +33,15 @@
           <span class="icon-bar"></span>
           <span class="icon-bar"></span>
         </button>
+{%- block navbar_brand %}
```
Comment (jirikuncar) [inline invenio/base/templates/header_base.html]: can you keep the indentation of Jinja2 blocks with HTML?

Comment (greut) [inline invenio/base/templates/header_base.html]: sure.

File: invenio/base/templates/header_base.html [hunk 2]
```diff
@@ -24,83 +24,91 @@
...
+          </li>
+        {%- else -%}
+          <li><a href="{{ item.url }}">{{ item.text|safe }}</a></li>
+        {%- endif -%}
+      {%- endfor -%}
+      </ul>
+      {%- endblock %}
+      <ul class="nav navbar-nav navbar-right">
+      {% if current_user.is_guest %}
+        <li>
```
Comment (greut) [inline invenio/base/templates/header_base.html]: This `<li>` was missing.

**Comment by jirikuncar** [comment]
@greut is it ready?

**Comment by greut** [comment]
@jirikuncar rebased and ready.

**Comment by jirikuncar** [comment]
@greut `3: M190 line is too long (73 > 72)` + `s/More blocks/Adds more blocks/`

**Comment by greut** [comment]
@jirikuncar thanks.

**Pull request merged**